### PR TITLE
Add Costa（光谷世界城）

### PR DESCRIPTION
**Pull request by lvwzhen**

### CODE REVIEW

File: wuhan.geojson [hunk 1]
```diff
@@ -1,45 +1,80 @@
 {
```
Comment (xhacker) [inline wuhan.geojson]: Can you use the existing format (2 spaces tab, `{` on the new line)? Thanks!

Comment (sinchang) [inline wuhan.geojson]: maybe should add .editorconfig to repo. 😁

Comment (xhacker) [inline wuhan.geojson]: You can make a pr if you like :)

Comment (lvwzhen) [inline wuhan.geojson]: OK

**Pull request closed without merging**